src/System.Runtime.Extensions/pkg/unix/System.Runtime.Extensions.pkgproj

```diff
@@ -11,6 +11,11 @@
     <ProjectReference Include="..\..\src\System.Runtime.Extensions.csproj">
       <OSGroup>Unix</OSGroup>
     </ProjectReference>
+    <!-- AOT implementation comes from AOT package -->
+    <File Include="$(PlaceholderFile)">
+      <TargetPath>runtimes/aot/lib/uap10.1</TargetPath>
```

Comment:
It does not make sense to have anything that has to do with uap101 in Unix package.